tests/Functional/RepositoryProxyTest.php

```diff
@@ -93,6 +93,7 @@ public function assertions(): void
     /**
      * @test
      * @group legacy
+     * @noinspection PhpDeprecationInspection
```

Comment:
not sure we should add PHPStorm specific annotations? :thinking: 

I'm not even sure we should hide that these methods use deprecated methods...

Reply:
I have reverted it